### PR TITLE
feat: add sharing instructions and policy refresh

### PR DESCRIPTION
**Pull request by jacobdadams**
Ok, after much rebasing and re-editing here's a fresh look at the reworked sharing and policy pages as part of https://github.com/agrc/porter/issues/87. 

Goals:

- Guide first-time and veteran contributors alike through the sharing process.
- Establish clear data lifecycle policies for how and why we do things.
- Continue to clarify the interactions between the different parts of the SGID.
- Lay the policy foundation for a proper db(s) for metadata and stewardship info (https://github.com/agrc/porter/issues/92)

### CODE REVIEW

File: about/policy/index.md
Comment (steveoh) [inline about/policy/index.md]: **question**: did this get moved or is it a bad rebase?

**Review by steveoh — COMMENTED** [review]
looks good. I'm excited to get this out there. I think once the question is answered we are good to go unless anyone else has concerns.

File: about/policy/index.md
Comment (jacobdadams) [reply about/policy/index.md]: The order got moved around in another branch and pushed to master. I rebased at some point after that and probably mixed it up there. I've checked the other pages/changes and they look good, so I think that's the only snafu (famous last words...)

Comment (jacobdadams) [reply about/policy/index.md]: And yes, I want to run this by the larger group as well given the amount of changes and the very specific policy wording. I think everyone's already seen the first version, but it's been long enough that a fresh look could prove useful.

File: about/policy/metadata/index.md
Comment (nathankota) [inline about/policy/metadata/index.md:26]:
```suggestion
Tags are used by ArcGIS Online in its search algorithms and can be used in Open Data to group data. At a minimum, each SGID dataset's tags should include the stewarding agency (ie: UGRC), "SGID", and the appropriate category name. We also recommend adding any additional, relevant tags that might help with data discoverability. However, there is no need to include words that are already contained in the layer's title (in fact, these will be omitted).
```

Comment (nathankota) [reply about/policy/metadata/index.md:26]: not a critical suggestion, so please feel free to ignore

File: about/policy/metadata/index.md
Comment (nathankota) [inline about/policy/metadata/index.md:37]: **question:** should there be a link to the open data about page for this data set, to show its summary? (https://opendata.gis.utah.gov/datasets/utah::utah-public-libraries-2/about)

File: about/policy/metadata/index.md
Comment (nathankota) [inline about/policy/metadata/index.md:44]:
```suggestion
The first line of the description should be the date the dataset was last updated so people have a clear and consistent place to see how fresh or stale it is.
```

File: about/policy/metadata/index.md
Comment (nathankota) [inline about/policy/metadata/index.md:53]: **question:** does the example need to include a contact or link to the contact info section like it had before?

File: sharing/examples/index.md
Comment (nathankota) [inline sharing/examples/index.md:29]:
```suggestion
Sometimes a project needs a 10,000-foot overview of land ownership, especially in Utah where large amounts of land are owned by private individuals, the state, and the federal government. Utah has tasked the State Institutional Trust Lands Administration (SITLA) with maintaining a state-wide [land ownership layer]({% link data/cadastre/land-ownership/index.html%}#LandOwnership) as part of their responsibilities, which they share on their [website](https://trustlands.utah.gov/resources/maps/). Because this is such an important and fundamental dataset, SITLA also shares the data with the Open SGID and the SGID Open Data site for easy access to all.
```

File: sharing/index.md
Comment (nathankota) [inline sharing/index.md:20]:
```suggestion
Nearly all the data that make the SGID such a valuable resource come from cities, counties, and other state agencies. We offer a heartfelt **Thank You!** on behalf of both UGRC and all the users of the SGID to everyone who has contributed data.
```

File: sharing/metadata/index.md
Comment (nathankota) [inline sharing/metadata/index.md:55]: Maybe readers won't know or care what "DTS" means?
```suggestion
_Note: If you contribute data to Utah's State Geographic Information Database (SGID), we encourage you to use our [metadata form](https://docs.google.com/forms/d/e/1FAIpQLSeJSV935GTN3djtErkBdeXkumuZvd10z2805It1-eFn0Mlo9w/viewform?usp=sf_link). It's a great way to update your metadata and make it more discoverable. The form covers all the questions above and—best of all—after you submit your responses, your portion of the process is done! UGRC, in collaboration with a professional editor colleague, will handle the remaining details._
```

File: sharing/open-sgid/index.md
Comment (nathankota) [inline sharing/open-sgid/index.md:47]: **question:** Does this conflict with the [second qualification](https://deploy-preview-1784--gis-utah-gov.netlify.app/about/policy/sgid/#open-sgid-qualifications)?

**Review by nathankota — COMMENTED** [review]
I added some mostly benign comments and questions, but serious thanks and praises for your assessment of all the SGID parts, pieces, policies, guidelines and your preparation of PR's. 

image

**Comment by steveoh** [comment]
Is this week a good target to merge?

**Comment by jacobdadams** [comment]
I got caught up with sendgrid and supervisor. Yes, I'll try to get to this
in the next day or two.

File: about/policy/metadata/index.md
Comment (jacobdadams) [reply about/policy/metadata/index.md:37]: I'm going to say no- the example text here is self-contained and won't break if that layer is updated, disappears, etc.

File: about/policy/metadata/index.md
Comment (jacobdadams) [reply about/policy/metadata/index.md:53]: Let's add this to a list of things to look at after we merge this. This page may need a little more attention.

File: sharing/metadata/index.md
Comment (jacobdadams) [reply sharing/metadata/index.md:50]: Again, leaving this to a later review (I didn't look over the pre-existing metadata stuff as much as the other stuff, it just got split into two places).

File: sharing/open-sgid/index.md
Comment (jacobdadams) [reply sharing/open-sgid/index.md:47]: it does, but the qualification uses `should` to cover the opportunity for exceptions. Mainly this is targeted to situations where we are updating the source of an existing layer to a new AGOL/Enterprise service (ala OGM or geology stuff).

**Review by jacobdadams — COMMENTED** [review]
Ok @steveoh , one last question before we merge. Months ago I did a quick-and-dirty addition of the "Sharing" tab to the top navigation right after "Developer." Are there additional things we need to do to make that cleaner/more persistent (I'm remembering something about the colors underneath the "active" tab?) or should I just strip that for now and find somewhere else to link the `sharing/` page from?

**Comment by steveoh** [comment]
> Ok @steveoh , one last question before we merge. Months ago I did a quick-and-dirty addition of the "Sharing" tab to the top navigation right after "Developer." Are there additional things we need to do to make that cleaner/more persistent (I'm remembering something about the colors underneath the "active" tab?) or should I just strip that for now and find somewhere else to link the `sharing/` page from?

Let's strip it and put it somewhere else. The nav has mobile issues already and this addition makes it worse.

**Comment by jacobdadams** [comment]
Ok, who wants to be the magic approving review?

**Comment by steveoh** [comment]
> Ok, who wants to be the magic approving review?

me

**Review by steveoh — DISMISSED** [review]

**Review by steveoh — APPROVED** [review]

**Pull request merged**
